proseco/guide.py

```diff
@@ -288,6 +290,26 @@ def process_include_ids(self, cand_guides, stars):
 
         super().process_include_ids(cand_guides, stars[ok])
 
+    def get_candidates_mask(self, stars):
+        """Get base filter for acceptable candidates.
```

Comment:
Worth noting that, in general, I don't know what value the label of "acceptable" really has when the CCD-position and more detailed spoiler checks have not been done.  But that's more of an issue for the seeking-the-right-roll PR.

Reply:
"Acceptable" means it *can* be chosen as a candidate.  It doesn't mean it *will* be a candidate.  Conversely, a star that is "not acceptable" will never be chosen as a guide or acq, so this is the initial entry ("base") filter.